use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -38,9 +35,181 @@ console.
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
+## Automate tiering with policies
+
+A tiering policy automatically moves data to the object storage tier. Any chunks that only contain data
+older than the `move_after` threshold are moved. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. You can add tiering policies to hypertables, including continuous aggregates.
+
+A tiering policy schedules a job that runs periodically to migrate eligible chunks. The migration is asynchronous. The chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.
+
+### Add a tiering policy
+
+To add a tiering policy, call `add_tiering_policy`:
+
+```sql
+SELECT add_tiering_policy(hypertable REGCLASS, move_after INTERVAL, if_not_exists BOOL = false);
+```
+
+For example, tier chunks that are more than three days old in the `example` hypertable in the following way: 
+
+```sql
+SELECT add_tiering_policy('example', INTERVAL '3 days');
+```
+
+### Remove a tiering policy
+
+To remove an existing tiering policy, call `remove_tiering_policy`:
+
+```sql
+SELECT remove_tiering_policy(hypertable REGCLASS, if_exists BOOL = false);
+```
+
+For example, remove the tiering policy from the `example` hypertable in the following way:
+
+```sql
+SELECT remove_tiering_policy('example');
+```
+
+If you remove a tiering policy, new scheduled chunks will not be tiered. However, already tiered chunks won't be untiered. You can [untier chunks manually](#manually-tier-and-untier-chunks) to the local storage.
+
+## Manually tier and untier chunks
+
+If tiering policies do not meet your current needs, you can tier and untier chunks manually. 
+
+### Tier chunks
+
+Tiering a chunk is an asynchronous process that schedules the chunk to be tiered. In this example, you use a hypertable called `example` and tier chunks older than three days. You then proceed to list tiered chunks.
+
+<Procedure>
+
+1. At the psql prompt, select all chunks in the table `example` that are older
```

Comment:
Connect to your service using console or psql

or something along those lines. 

example hypertable